scripts/logs/README.md

```diff
@@ -0,0 +1,48 @@
+# How to follow log outputs
```

Comment:
Shouldn't we just point to `tkn` instead ?

Reply:
this is copied straight from pipeline - we might do something different eventually - but I wanted to have a smooth migration, by using the existing scripts mostly wrapped into tasks

Reply:
I've seen your PR about this, once it's done on pipelines I can port it to plumbing if that's ok.